source/partial-deep.d.ts

```diff
@@ -10,6 +10,13 @@ export type PartialDeepOptions = {
 	@default false
 	*/
 	readonly recurseIntoArrays?: boolean;
+
+	/**
+	Whether to authorize undefined values in arrays.
+
+	@default true
+	*/
+	readonly allowUndefinedInArrays?: boolean;
```

Comment:
Should be `allowUndefinedInNonTupleArrays` since it does not apply to tuples and the above option using `*array` in the name does.

Reply:
Yes you are right